.env.example

```
@@ -128,6 +134,11 @@ GOOGLE_ENABLED=false
 #SMTP_IGNORE_TLS=false
 #SMTP_REQUIRE_TLS=false
 
+# If your SMTP server/relay does not have SMTP AUTH,
+# IE treats all local connections as already authenticated.
+# Then set this to false, default is true you do want SMTP AUTH.
+#SMTP_AUTH_REQ=true
```

This name is confusing, considering that it's right under `SMTP_REQUIRE_TLS` there is a bias to read this as `SMTP_AUTH_REQUIRE`, which isn't the intended meaning.

Should be renamed for clarity

```suggestion
#SMTP_AUTH_REQUEST=true
```

---

I was thinking the same, but also was trying to avoid long variable names.